config/config.go

```diff
@@ -36,6 +36,10 @@ type Config struct {
 	DaemonPollInterval int `json:"daemon_poll_interval"`
 	// BranchPrefix is the prefix used for git branches created by the application.
 	BranchPrefix string `json:"branch_prefix"`
+	// WorktreePattern is the pattern used to determine worktree location.
+	// Supports variables: {repo_root}, {repo_name}, {issue_number}, {title}, {timestamp}
+	// Example: "{repo_root}/worktree/{issue_number}-{title}"
+	WorktreePattern string `json:"worktree_pattern"`
```

Comment:
I think it would be good to make this apply to the branch name too. I think it's good for branch names to be 1:1 with worktree paths.

Maybe we can call it `WorkStreamPattern`?

Reply:
Are you thinking the branch name would be another pattern we include?